assets/js/src/newsletter_editor/components/history.js

```diff
@@ -29,6 +41,26 @@ Module.HistoryView = Marionette.View.extend({
     this.model.states.unshift(json.body);
     this.model.currentStateIndex = 0;
     this.model.states.length = Math.min(this.model.states.length, this.MAX_HISTORY_STATES);
+    this.updateArrowsUI();
+  },
+
+  canUndo: function canUndo() {
+    return this.model.currentStateIndex < (this.model.states.length - 1)
+  },
+
+  canRedo: function canRedo() {
+    return this.model.currentStateIndex !== 0;
+  },
+
+  updateArrowsUI: function updateArrowsUI() {
+    this.elements.undo.addClass('mailpoet_history_arrow_inactive');
+    this.elements.redo.addClass('mailpoet_history_arrow_inactive');
+    if (this.canUndo()) {
+      this.elements.undo.removeClass('mailpoet_history_arrow_inactive');
+    };
+    if (this.canRedo()) {
+      this.elements.redo.removeClass('mailpoet_history_arrow_inactive');
+    };
```

Comment:
Even without jQuery this could just be:
```
this.elements.undo.classList.toggle('mailpoet_history_arrow_inactive', !this.canUndo());
this.elements.redo.classList.toggle('mailpoet_history_arrow_inactive', !this.canRedo());
```